src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -520,33 +527,198 @@ export const concatAttachmentsAndResponses = (
   return response
 }
 
-export const maskUidOnLastField = (
-  autoReplyData: EmailAutoReplyField[],
-): EmailAutoReplyField[] => {
+/**
+ * Function to generate email data
```

Comment:
saying "Function to" every single time gets quite repetitive. you can just say "Generates email data for a single field".

Reply:
also, could you add docs for the parameters of this function? the generics + `getFormattedFunction` would probably be quite confusing to anyone without context